platform-sdk/docs/core/dynamic-address-book/dynamic-address-book.md

```diff
@@ -0,0 +1,183 @@
+see an event in round B, in which case it belongs to round B+1_. In the diagram below, all the nodes labeled `R1`
+can strongly see one or more nodes labeled `R0`, all nodes labeled `R2` can strongly see one or more nodes labeled
+`R1`, and so on. All nodes labeled `R0` belong to round 0. All nodes labeled `R1` belong to round 1, and so on.
+
+![Events Belong To Rounds](./HashgraphRound.drawio.svg)
+
+To determine the round in which the network receives an event, we must define a few more properties. The first event
+from each node in a round is called a `witness`. A witness in round R is called `famous` if, for a
+super majority of nodes, it can be strongly seen by a witness in a later round R+d, d > 0. The process of
+determining fame is called `voting`. Witnesses in later rounds vote on the fame of witnesses in earlier rounds. An
+honest node will have at most one witness in each round, making it a unique witness for the node in that round.
+If a dishonest node creates multiple witnesses, one is chosen to be considered the `unique famous witness` for
+that node for that round.  _An event X is `received by the network` in round R if R is the least round such that X
+is an ancestor of all the unique famous witnesses in round R_.  **Note:** The round B that an event belongs to is
+necessarily less than the round R in which it is received by the network and given a consensus timestamp and order.
+The diagram does not depict what round the events are received, just the rounds to which the events belong.
+
+In our code, all events which have been received by the network in the same round are bundled into the same
+`ConsensusRound` object. The addition of a single event to the hashgraph can cause multiple rounds to have their
+unique famous witnesses determined, which may cause one or more prior rounds to come to consensus,
+producing a sequence of `ConsensusRound` objects for post-consensus handling. Each `ConsensusRound` object
+contains the effective `consensus roster` that determined consensus on that round. Only the nodes in the 
+effective consensus roster's witnesses can vote on the fame of prior witnesses.  (?? This last sentence needs to be 
+verified ??)
+
+When receiving events through gossip, we need to validate the events with respect to the  effective consensus roster 
+when the event was created. Let R be the last round to come to consensus. We define the `platformRound` to be R+1, 
+or the next round the platform is working towards consensus on. Consensus rosters are created in the handling of 
+transactions in one round and become effective on the platform in a later round. The `rosterCreationRound`
+is the `ConsensusRound` that created the roster. The `rosterEffectiveRound` is the `platformRound` that the
+consensus roster becomes effective on. There is a minimum `rosterEffectiveOffset` which dictates the minimum number
```

Comment:
>We define the `platformRound` to be R+1, or the next round the platform is working towards consensus on.

>The `rosterEffectiveRound` is the `platformRound` that the consensus roster becomes effective on.

Together, these two statements are confusing. If the platformRound is the next round that has not yet reached consensus, how do we know a roster will become effective on that round (the very next round to reach consensus?)

Reply:
I think if the term `platformRound` is removed, this is easier to understand.